qiskit/tools/events/__init__.py

```diff
@@ -10,7 +10,24 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Events
+"""
+===================================
+Events (:mod:`qiskit.tools.events`)
+===================================
+
+A helper component for publishing and subscribing to events
```

Comment:
While we're fixing the docs... 🙂 
```suggestion
A helper component for publishing and subscribing to events.
```